src/artery/CMakeLists.txt

```diff
@@ -182,4 +182,8 @@ if(NOT is_multi_config)
     install(
         PROGRAMS ${PROJECT_BINARY_DIR}/${CMAKE_CFG_INTDIR}/run_artery.sh.install
         DESTINATION ${CMAKE_INSTALL_BINDIR} RENAME run_artery.sh)
+
+    if(NOT CMAKE_BUILD_TYPE OR "${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
```

Comment:
I have one wish remaining: Add a CMake option controlling if this feature is enabled at all. A possible name might `VSCODE_LAUNCH_INTEGRATION` but I am open to suggestions here. Possibly some people might not like it if their *launch.json* is touched by us ;-)

Reply:
True. I've added the `VSCODE_LAUNCH_INTEGRATION` option and `cmake` now behaves nicely now. It will create or extend an already present `launch.json` when Artery is built as `Debug`.